website/pages/en/versions.js

```diff
@@ -0,0 +1,109 @@
+        props.language + '/' : '';
+    return (
+        <div className="docMainWrapper wrapper">
+        <Container className="mainContainer versionsContainer">
+          <div className="post">
+            <header className="postHeader">
+              <h1>{title} Versions</h1>
+            </header>
+            <h3 id="latest">Current version (Stable)</h3>
+            <p>Latest version of KaTeX.</p>
+            <table className="versions">
+              <tbody>
+                <tr>
+                  <th>{latestVersion}</th>
+                  <td>
+                    <a
+                      href={`${baseUrl}docs/${language}node.html`}>
+                      Documentation
+                    </a>
+                  </td>
+                  <td>
+                    <a href={`${repoUrl}/releases/tag/v${latestVersion}`}>
+                      Release Notes
+                    </a>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <h3 id="rc">Latest Version</h3>
+            Here you can find the latest documentation and unreleased code.
+            <table className="versions">
+              <tbody>
+                <tr>
+                  <th>master</th>
+                  <td>
+                    <a
+                      href={`${baseUrl}docs/${language}next/node.html`}>
+                      Documentation
+                    </a>
+                  </td>
+                  <td>
+                    <a href={repoUrl}>Source Code</a>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <h3 id="archive">Past Versions</h3>
+            <p>
+              Here you can find documentation for previous versions of KaTeX.
+            </p>
+            <table className="versions">
+              <tbody>
+                {versions.map(
+                  version =>
+                    version !== latestVersion && (
+                      <tr key={version}>
+                        <th>{version}</th>
+                        <td>
+                          <a
+                            href={`${baseUrl}docs/${
+                              language
+                            }${version}/node.html`}>
+                            Documentation
+                          </a>
+                        </td>
+                        <td>
+                          <a href={`${repoUrl}/releases/tag/v${version}`}>
+                            Release Notes
+                          </a>
+                        </td>
+                      </tr>
+                    )
+                )}
+              </tbody>
+            </table>
+            <p>
+              You can find past versions of KaTeX on{' '}
+              <a href={`${repoUrl}/releases`}>GitHub</a>.
+            </p>
+          </div>
+        </Container>
+      </div>
+    );
+}
+
+Versions.title = 'Versions';
+
+module.exports = Versions;
```

Comment:
I didn't see this in the netlify preview, I guess because there's no `versions.json` yet.  How should I test this?

Reply:
@edemaine Run 
```bash
cd website
yarn version 1.0.0
```

Reply:
I get the following (first half looks good, second half looks bad):

```sh
$ yarn version --new-version 0.10.0-beta
yarn version v1.9.2
warning package.json: No license field
warning package.json: No license field
info New version: 0.10.0-beta
$ docusaurus-version
No version number specified!
Pass the version you wish to create as an argument.
Ex: 1.0.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/version for documentation about this command.
```

Reply:
@edemaine `--new-version` is not needed for `version`.

Reply:
@edemaine Yarn has built-in version command: https://yarnpkg.com/en/docs/cli/version. It should be `yarn run version`, sorry.

Reply:
Ahh, got it now, thanks.

Reply:
@edemaine Also the `version` script doesn't run `prestart`, so it will probably give an error about `cli.md` not found. I'll commit the changes.